src/components/form-attachment/list.vue

```diff
@@ -482,6 +505,13 @@ export default {
       "readError": "Something went wrong while reading “{filename}”.",
       "success": "{count} file has been successfully uploaded. | {count} files have been successfully uploaded.",
       "link": "Dataset linked successfully."
+    },
+    "datasetsPreview": {
+      // This is a title shown above a section of the page.
+      "title": "Datasets Preview",
```

Comment:
To avoid this being translated twice (here and in `FormDraftTesting`), could you move this message to en.json5 and remove it from `FormDraftTesting`? Maybe it could be `common.datasetsPreview`.